CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ project(smack)
 
 if (NOT WIN32 OR MSYS OR CYGWIN)
 
-  file(STRINGS "bin/versions" LLVM_VERSION_STR REGEX "LLVM_SHORT_VERSION=[0-9]+")
+  file(STRINGS "bin/versions" LLVM_VERSION_STR REGEX "LLVM_SHORT_VERSION=\"[0-9]+\"")
```

Comment:
How is this change related? is it because workflows run with a different shell or version of CMake?

Reply:
No, we used to have assignments like `A=a` in `bin/versions` but now they are changed to `A="a"` so the old regex didn't work. It seems the variables in the CMakefil were still set correctly on Travis CI VMs but not on GitHub actions.